app.js

```diff
@@ -35,31 +37,23 @@ app.use(function(req, res, next) {
   next(err);
 });
 
-// error handlers
-
-// development error handler
-// will print stacktrace
-if (app.get('env') === 'development') {
-  app.use(function(err, req, res, next) {
-    res.status(err.status || 500);
-    res.render('error', {
-      message: err.message,
-      error: err,
-    });
-  });
-}
-
-// production error handler
-// no stacktraces leaked to user
+// error handler
 app.use(function(err, req, res, next) {
+  console.error(err);
   res.status(err.status || 500);
   res.render('error', {
     message: err.message,
-    error: {},
+    error: err,
   });
 });
 
-if (process.env.NODE_ENV.toLowerCase() !== 'test' && !!process.env.CI)
+if ((process.env.NODE_ENV || '').toLowerCase() !== 'test' && !!process.env.CI)
   scheduler.start();
 
+app.listen(cfg.port, function() {
+  console.log(
+    `Starting sample-appointment-reminders at http://localhost:${cfg.port}`
+  );
+});
```

Comment:
This neeeds to be moved into the if statement otherwise the tests will run forever